_content/av/transnationalizing-buddhism_jones.eric.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Transnationalizing Buddhism with Dr. John Marston"
+authors:
+  - "Eric Jones"
```

Comment:
Put John Marston as the author and move Eric Jones to editor:  that's how I usually tag interviews like this 😁